.travis.yml

```diff
@@ -8,7 +8,7 @@ install:
 # avoid default npm install
 - true
 script:
-- "./launch -d -n update"
+- "./launch -bpe -d -n update"
```

Comment:
I'd prefer more readable form "--bypass_emscripten" instead of -bpe.

Reply:
done, now full convention is used for all options